.github/workflows/docs-check.yml

```diff
@@ -1,4 +1,4 @@
-name: Docs
+name: Docs Check
```

Comment:
**issue**: Changing this calls for changing the job names per our naming convention. 

I've referenced our naming convention in other PRs, but having it in the style guide is probably good. While it is not there yet, this PR adds it and can be a reference: #69.